tools/validators/instance_validator/tests/fake_instances/GOOD/translation_string_states.yaml

```diff
@@ -20,7 +20,7 @@
 DMP_EDM-17-GUID:
   type: HVAC/DMP_EDM
   code: DMP_EDM-17
-  cloud_device_id: "foobar"
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.